### PR TITLE
Ensar branch  2

### PR DESCRIPTION
**Pull request by Zorlu-Test**
Emsar branch class added

### CODE REVIEW

File: src/test/java/BaseClass.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public static int allNumberTotal() {
             sum += number;
         }
         System.out.println("max = " + max);
+        System.out.println("MAX = " + max);
```
Comment (ademcigdem) [inline src/test/java/BaseClass.java:24]: This is nice to see

**Review by AndrianHrisotimos — CHANGES_REQUESTED** [review]
I wanted to add my main method as well

**Review by ecaliskanunal — APPROVED** [review]

**Comment by ecaliskanunal** [comment]
Marvelous!

**Review by busragokner — COMMENTED** [review]
Good job

**Review by fikretellek — APPROVED** [review]

**Comment by CengizKoch** [comment]
good

**Review by AndrianHrisotimos — APPROVED** [review]

**Comment by GokceKochan** [comment]
done

**Pull request merged**